testnets/_IBC/composabletestnet-osmosistestnet

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "composabletestnet",
+    "client_id": "07-tendermint-23",
+    "connection_id": "connection-18"
+  },
+  "chain_2": {
+    "chain_name": "osmo-test-5",
```

Comment:
chain_name for osmo-test-5 is osmosistestnet5

Reply:
updated thanks!